locale/en-us/stop.dialog

```diff
@@ -0,0 +1 @@
+Do you want me to stop mall parsing?
```

Comment:
I think "Do you want to exit the mall directory?" would be a more user-friendly way of asking